packages/preset-rules/src/customRules.ts

```diff
@@ -4,36 +4,25 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import * as fs from 'fs';
-import path from 'path';
 import axe from 'axe-core';
-import { processFiles } from '@sa11y/common';
-
-// Directory containing custom rules and checks
-const customRulesPathDir = 'custom-rules';
+import checkData from './custom-rules/checks';
+import rulesData from './custom-rules/rules';
+import changes from './custom-rules/changes';
 
 export function registerCustomRules(): void {
-    const directoryPath = path.join(__dirname, customRulesPathDir);
     const newChecks: axe.Check[] = [];
     const newRules: axe.Rule[] = [];
 
-    try {
-        // Path to the JSON file with changes to existing rules
-        const changesInExistingRulesPath = path.join(directoryPath, 'changes.json');
-        // Read and parse existing rule changes
-        const existingRuleData = fs.readFileSync(changesInExistingRulesPath, 'utf-8');
-        const { rules } = JSON.parse(existingRuleData) as { rules: axe.Rule[] };
-        if (rules && Array.isArray(rules)) {
-            newRules.push(...rules);
-        }
+    // Read and parse existing rule changes
+    const { rules } = changes as { rules: axe.Rule[] };
+    const newRulesData = rulesData as axe.Rule[];
+    const newChecksData = checkData as axe.Check[];
```

Comment:
it's better we assign types at `rules.ts`, `checks.ts` & `changes.ts`  than inferring them here using `as` keyword, that way we can impose type check when adding new rules into ts file.

Reply:
Added.